include/mbedtls/config.h

```diff
@@ -694,6 +694,9 @@
  * for too long on ECC (and, hence, X.509 or SSL/TLS) operations.
  *
  * Uncomment this macro to enable restartable ECC computations.
+ *
+ * \note  MBEDTLS_ECP_RESTARTABLE cannot be defined if there is an alternative
```

Comment:
Avoid using “cannot” to mean “if you do this, something will break”. Use “must not” or “do not”. Suggestion:

> This option only works with the default software implementation of elliptic curve functionality. It is incompatible with `MBEDTLS_ECP_ALT`, `MBEDTLS_ECDH_xxx_ALT` and `MBEDTLS_ECDSA_xxx_ALT`.